src/py/flwr/superexec/deployment.py

```diff
@@ -33,21 +34,37 @@
 
 
 class DeploymentEngine(Executor):
-    """Deployment engine executor."""
+    """Deployment engine executor.
+
+    superlink: str (Default: "0.0.0.0:9091")
+        Address of the `SuperLink` to connect to.
+    root_certificates: Optional[str] (Default: None)
+        Specifies the path to the PEM-encoded root certificate file for
+        establishing secure HTTPS connections.
+    flwr_dir: Optional[str] (Default: None)
+        The path containing installed Flower Apps.
+    """
 
     def __init__(
         self,
-        address: str = DEFAULT_SERVER_ADDRESS_DRIVER,
-        root_certificates: Optional[bytes] = None,
+        superlink: str = DEFAULT_SERVER_ADDRESS_DRIVER,
+        cert_path: Optional[str] = None,
+        flwr_dir: Optional[str] = None,
```

Comment:
We could also think about a constructor that doesn't take any arguments, and defer setting these values until a new `Executor` method (`.configure(self, executor_config: Dict[...])`?) is called that takes the parsed `--executor-config` as an argument